claim-db-worker/app/web/schema/page.tsx

```
@@ -0,0 +1,329 @@
+        {errorDetails && (
+          <div className="bg-step p-4 rounded border border-subtle mb-6">
+            <pre className="text-sm text-muted whitespace-pre-wrap break-words">
+              {errorDetails}
+            </pre>
+          </div>
+        )}
+        <div className="flex gap-3 justify-end">
+          <button
+            onClick={() => setShowErrorModal(false)}
+            className="px-4 py-2 text-muted hover:text-white transition-colors"
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)